src/services/navigation.js

```diff
@@ -0,0 +1,5 @@
+import _ from 'lodash';
+
+export function getParams(navigation) {
+  return _.get(navigation, 'state.params');
```

Comment:
you can say `_.get(navigation, 'state.params', {});`
This last parameter is a default value, in case that navigation.state.params is undefined.

This is useful (to return the empty object for default) so that you can always deconstruct stuff from navigationParams without worrying whether they exist.

If you say `const { xy } = navigationParams` and for some reason navigationParams is `undefined`, your app will crash.